generators/app/templates/src/app/components/Routes/constants.ts

```diff
@@ -0,0 +1,32 @@
+import { lazy } from 'react';
+import i18next from 'i18next';
+
+import { Nullable } from '~utils/types';
+import { User } from '~app/reducer';
+
+import PATHS from './paths';
+
+const Home = lazy(() => import('../../screens/Dashboard'));
+const Login = lazy(() => import('../../screens/Login'));
+
+const MAIN_PUBLIC_PATH = PATHS.login;
+const MAIN_PRIVATE_PATH = PATHS.home;
+
+export const ROUTES = [
+  {
+    exact: false,
+    path: PATHS.login,
+    component: Login,
+    title: i18next.t('Routes:loginTitle'),
+    description: i18next.t('Routes:loginDescription'),
+    redirectTo: (user: Nullable<User>) => (user ? MAIN_PRIVATE_PATH : undefined)
```

Comment:
Okay, it seems you want to handle this here. Why?
If you have 500 routes this becomes unmaintainable

Reply:
We could create a common utility function to validate these things, but it would mean calling it over and over again, when including this behaviour in the Route component would need no maintainance. Maybe I'm missing something? 
Let me know what you think

Reply:
you can return `redirectTo: () => {}` if you wanted to. I don't see why that would not be maintainable. Previously we had a gigantic list of routes in the Routes component, it's kind of the same, right? We need to have this information somewhere